cli/src/test/kotlin/uk/gov/justice/digital/cli/client/BlockingDomainClientTest.kt

```diff
@@ -134,6 +141,11 @@ class BlockingDomainClientTest {
                     @Suppress("UNUSED_PARAMETER") limit: Int): HttpResponse<List<List<String?>>> {
             return HttpResponse.ok(domainPreviewData)
         }
+        @Post("/publish", consumes = [MediaType.APPLICATION_JSON], produces = [MediaType.APPLICATION_JSON])
+        fun preview(@Suppress("UNUSED_PARAMETER") domainName: String,
```

Comment:
Minor: The function name should be publish instead of preview

Reply:
Argh! Good spot. I have a feeling the approvals will get cleared if push a rename. I guess it'll have to be left as an exercise for the reader. :(